.github/workflows/release.yml

```diff
@@ -25,9 +25,8 @@ jobs:
 
       - name: Semantic release
         run: |
-          source ~/.nix-profile/etc/profile.d/nix.sh
-          nix develop --command npm install semantic-release
-          TEST_TMPDIR=$HOME/.bazel_test_opt nix develop --command npx semantic-release
+          /usr/bin/nix develop --command npm install semantic-release
```

Comment:
/usr/bin should be in PATH. You can probably just use `nix`

Reply:
Updated, thank you.